Compiler/script/cc_symboltable.cpp

```diff
@@ -140,107 +150,112 @@ void symbolTable::reset() {
     add_ex("noloopcheck", SYM_LOOPCHECKOFF, 0);
     add_ex("builtin", SYM_BUILTIN, 0);
 }
-int SymbolTableEntry::operatorToVCPUCmd() {
+int SymbolTableEntry::operatorToVCPUCmd()
+{
     //return ssize + 8;
     return vartype;
 }
-int symbolTable::find(const char*ntf) {
+
+int symbolTable::find(const char*ntf)
+{
     return symbolTree.findValue(ntf);
 }
 
-std::string symbolTable::get_friendly_name(int idx) {
-    std::string result;
-
-    idx &= STYPE_MASK;
-    if(idx > -1 && idx < entries.size()) {
-        result = entries[idx].sname;
-
-        if (idx & STYPE_POINTER) {
-            result = result + std::string("*");
-        }
-
-        if (idx & STYPE_DYNARRAY) {
-            result = result + std::string("[]");
-        }
+std::string symbolTable::get_friendly_name(int idx)
+{
+    int short_idx = (idx & STYPE_MASK);
+    if (short_idx <= 0 || short_idx >= entries.size())
+    {
+        return std::string("(invalid symbol)");
+    }
 
-        if (idx & STYPE_CONST) {
-            result = std::string("const ") + result;
-        }
+    std::string result = entries[short_idx].sname;
 
-    }
-    else
-        result = std::string("(invalid symbol)");
+    if (idx & STYPE_POINTER)  result += std::string("*");
+    if (idx & STYPE_DYNARRAY) result += std::string("[]");
+    if (idx & STYPE_CONST)    result = std::string("const ") + result;
 
     return result;
 }
 
-std::string symbolTable::get_name_string(int idx) {
-	if (idx & STYPE_CONST) {
+std::string symbolTable::get_name_string(int idx)
+{
+    if (idx < 0) return std::string("((NoSymbol))");
+    if (idx & STYPE_CONST)
+    {
         idx &= ~STYPE_CONST;
-		return std::string("const ") + get_name_string(idx);
+        return std::string("const ") + get_name_string(idx);
     }
 
-    if (idx & STYPE_DYNARRAY) {
+    if (idx & STYPE_DYNARRAY)
+    {
         idx &= ~(STYPE_DYNARRAY | STYPE_POINTER);
-		return get_name_string(idx) + std::string("[]");
+        return get_name_string(idx) + std::string("[]");
     }
 
-    if (idx & STYPE_POINTER) {
+    if (idx & STYPE_POINTER)
+    {
         idx &= ~STYPE_POINTER;
-		return get_name_string(idx) + std::string("*");
+        return get_name_string(idx) + std::string("*");
     }
 
     return entries[idx].sname;
 }
 
-const char *symbolTable::get_name(int idx) {
-	if (nameGenCache.count(idx) > 0) {
-		return nameGenCache[idx];
-	}
+const char *symbolTable::get_name(int idx)
+{
+    if (nameGenCache.count(idx) > 0)
+    {
+        return nameGenCache[idx];
+    }
 
-	std::size_t actualIdx = idx & STYPE_MASK;
```

Comment:
there are tools like gitk that let you look at your commits locally, so you don't have to push to github to see that you introduced gratuitous whitespace changes.

Reply:
I think these could be intentional, fernewelten mentioned he was adjusting codestyle to match our convention.